internal/storage/v2/clickhouse/sql/embed.go

```diff
@@ -5,5 +5,20 @@ package sql
 
 import _ "embed"
 
-//go:embed spans_insert.sql
-var SpansInsert string
+//go:embed insert_span.sql
+var InsertSpan string
+
+//go:embed select_spans_by_trace_id.sql
+var SelectSpansByTraceID string
+
+//go:embed select_services.sql
+var SelectServices string
+
+//go:embed select_operations_all_kinds.sql
+var SelectOperationsAllKinds string
+
+//go:embed select_operations_by_kind.sql
+var SelectOperationsByKind string
+
+//go:embed create_schema.sql
+var CreateSchema string
```

Comment:
Besides the schema file the rest of embedding feel like an overkill. It would make for more readable code in the storage if sql was just inline. What benefits do we get from this indirection? If I'm reading storage code I am at least two clicks away from seeing sql query that backs a specific code.

Reply:
@yurishkuro So do we just want to embed create schema and have the others as constants? If we're embedding one of them, isn't it nice to just have them all in one place?

Reply:
yes

Reply:
done